icons/messages-square.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M14 9c0 1.1-.9 2-2 2H6l-4 4V5c0-1.1.9-2 2-2h8c1.1 0 2 .9 2 2v4z" />
+  <path d="M18 9h2c1.1 0 2 .9 2 2v10l-4-4h-6c-1.1 0-2-.9-2-2" />
```

Comment:
I think we could use a bit more of the canvas, the icons will look better on 100% scale. The message balloon looks a bit squished now.

Suggestion improvement (right): 
<img width="57" alt="image" src="https://user-images.githubusercontent.com/11825403/232326420-5fa241cc-9fcd-4245-8c8e-13ea3d3133be.png">


```suggestion
  <path d="M14 9a2 2 0 0 1-2 2H6l-4 4V4c0-1.1.9-2 2-2h8a2 2 0 0 1 2 2v5Z" />
  <path d="M18 9h2a2 2 0 0 1 2 2v11l-4-4h-6a2 2 0 0 1-2-2v-1" />
```